modules/local/fusioninspector/main.nf

```diff
@@ -2,8 +2,8 @@ process FUSIONINSPECTOR {
     tag "$meta.id"
     label 'process_high'
 
-    conda "bioconda::dfam=3.7 bioconda::hmmer=3.4 bioconda::star-fusion=1.13.0 bioconda::trinity=2.15.1 bioconda::samtools=1.19.2 bioconda::star=2.7.11b"
-    container 'docker.io/trinityctat/starfusion:1.13.0'
+    conda "bioconda::dfam=3.7 bioconda::hmmer=3.4 bioconda::star-fusion=1.7.0 bioconda::trinity=2.15.2 bioconda::samtools=1.21 bioconda::star=2.7.11b"
```

Comment:
the conda declaration could be with the `environmentl.yml` as we do in nf-core modules.

you can get it directly from the seqera containers :) they already provide it